opentelemetry-sdk/Cargo.toml

```diff
@@ -66,6 +66,7 @@ harness = false
 [[bench]]
 name = "metric_counter"
 harness = false
+required-features = ["metrics"]
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/benches/metric_counter.rs#L10 Could you fix this comment too?

Reply:
Done. Be aware you must activated the feature with --feature event if we defined it in the Cargo.toml. However, instead of a compilation error cargo gives a nice reminder to activate the feature. It fixes also the use case when we run all bench with cargo bench. :)